README.RU.md

```diff
@@ -0,0 +1,198 @@
+
+Последня таблица просто показывает свободное место на удаленых харвестерах/фермах
+
+В последней секции отображается журнал выполненных действий -- а именно запущенные 
+задания засева и архивации участков. Это одна из частей интерактивной утилиты, которая
+отслеживает состояние.  Постоянная запись выполненных действий отсутствует, поэтому, если
+вы запустите новый интерактивный сеанс plotman, этот журнал будет пуст.
+
+## Ограничения и проблемы
+
+Прототестировано только на Linux. 
+The system is tested on Linux only. Plotman должен быть портирован на другие
+платформы, но это еще не сделано. Существуют некоторые проблемы, связанные с вызовом
+linux-подобных консольных программ (например, запуск `df` через `ssh`, чтобы получить свободное
+пространство на удаленных машинах в каталогах архивов).
+
+Интерактивный режим использует библиотеку `curses` ... плохо. Нажатия
+клавиш не принимаются, изменение размера экрана не работает, а минимальный размер терминала
+довольно большой.
+
+Plotman предполагает, что все участки являются k32. Опять же, это просто нереализованный
+функционал.
+
+Многие функции неконсистентно поддерживаются либо в "интерактивном"
+режиме, либо в режиме командной строки.
+
+Есть еще много багов и что необходимо сделать.
+
+Plotman всегда будет искать файл `plotman.yaml` на вашем компьютере в
+расположении по умолчанию заданному вашей ОС. Чтобы создать файл `plotman.yaml`, выполните:
+```shell
+> plotman config generate
+```
+
+Чтобы найти текущее расположение вашего `plotman.yaml` и проверить его существование, выполните:
+```shell
+> plotman config path
+```
+
+([See also](https://github.com/ericaltendorf/plotman/pull/61#issuecomment-812967363)).
+
+## Установка
+
+Установка для Linux:
+
+1. Plotman предполагает, что в системе уже установлена [Chia](https://github.com/Chia-Network/chia-blockchain).
+   Активируйте среду окружения `chia` набрав
+   `source /path/to/your/chia/install/activate`.
+2. Затем установите Plotman используя следующую комманду:
+   ```shell
+    > pip install --force-reinstall git+https://github.com/ericaltendorf/plotman@main
+    ```
+3. Plotman будет искать `plotman.yaml` на вашем компьютере в папке по умолчанию
+   в зависимости от ОС. Чтобы создать такой файл `plotman.yaml` и отобразить его местоположение,
+   выполните следующую комманду:
+   ```shell
+   > plotman config generate
+   ```
+   Файл конфигурации по умолчанию, используемый в качестве отправной точки, находится [здесь](./src/plotman/resources/plotman.yaml)
+4. Вот и всё! Вы можете запустить Plotman набрав `plotman version` для проверки его версии.
+   Выполните `plotman --help` для изучения доступных комманд.
+
+### Примечание разработчикам:
+
+Если Вы форкнули Plotman, на шаге установки замените комманду на `pip install --editable .[dev]` из коревой папки проекта для установки *вашей* версии plotman с дополнительными функциями тестирования и разработки.
+
+### Примечание от переводчика:
```

Comment:
Overall a dumb translation (which is readable though) without thinking too much into it.

Reply:
I guess we need a real community discussion on how to translate or not terms like plotting, plot, farming, harvester, ... because they sound/read terrible in Russian.